src/components/molecules/UiLoader/UiLoader.stories.js

```diff
@@ -1,58 +1,75 @@
 import UiLoader from '@/components/molecules/UiLoader/UiLoader.vue';
```

Comment:
@ArkadiuszKrzymienski I made a few improvements in the stories. 

- on/of switch for loader
- content from design to test real use cases
- 
![Zrzut ekranu 2022-09-2 o 16 07 00](https://user-images.githubusercontent.com/12138170/188167988-044485b2-39e7-424e-8d29-f98d5dc96101.png)

Reply:
ok great thanks 🙂